src/Filter/StringFilter.php

```diff
@@ -58,6 +64,10 @@ public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $value
 
         if (StringOperatorType::TYPE_NOT_CONTAINS === $type) {
             $or->add($queryBuilder->expr()->isNull(sprintf('%s.%s', $alias, $field)));
+
+            if ('' === $value['value']) {
+                $or->add(sprintf('%s.%s <> :%s', $alias, $field, $parameterName));
+            }
```

Comment:
Why did you add this ? This doesn't seem necessary to me.

`NOT LIKE ''` is working

Reply:
`NOT LIKE ''` is not equal `<> ''`. The DB Query Optimizer should convert the first expression to the second, but that doesn't mean you should rely on it. It is better not to use `LIKE` where it is not needed.